fedimint-testing/src/btc/mod.rs

```diff
@@ -41,4 +41,12 @@ pub trait BitcoinTest {
 
     /// Waits till tx is found in mempool and returns the fees
     async fn get_mempool_tx_fee(&self, txid: &Txid) -> Amount;
+
+    /// Scans all transactions in all blocks
```

Comment:
Wait, what?

Reply:
I see. This comment might be just misplaced and should be on the impl?

Reply:
I see why that's confusing, thanks for pointing out. At the time it was true for all impls, but regardless doesn't belong in docstrings for a trait method.